pkg/testcase/upgradenodereplacement.go

```diff
@@ -33,7 +33,7 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, flags *customflag.FlagConfi
 	// create server names.
 	var serverNames, instanceServerIds, newExternalServerIps, newPrivateServerIps []string
 	for i := 0; i < len(cluster.ServerIPs); i++ {
-		serverNames = append(serverNames, fmt.Sprintf("%s-server-replace-%d", resourceName, i+1))
+		serverNames = append(serverNames, fmt.Sprintf("%s-distros-server-replace-%d", resourceName, i+1))
```

Comment:
https://github.com/rancher/distros-test-framework/pull/178/files#r1823336102 May be add a tag when creating new instance?